specs/BadgeNotifications/BadgeNotifications-spec.md

```diff
@@ -0,0 +1,192 @@
+|-|-|-|-|
+| UpdateBadge | Applies a change to the badge's glyph or number by providing a `BadgeNotification` object. This method is used to update the badge displayed on the app's tile to reflect a new status or count. | BadgeNotification `notification` | void |
+| ClearBadge | Removes the badge notifications for the app from the action center. This method is useful for clearing any badge notifications when they are no longer relevant or when the app wants to reset its badge status. | | Windows.Foundation.IAsyncAction |
+
+## BadgeGlyphValue enum
+
+The `BadgeGlyphValue` enum defines a set of predefined glyphs that can be used to represent various statuses or notifications on an application's badge. These glyphs provide a visual cue on the app's tile, allowing users to quickly understand the app's status or to be alerted to new information without opening the app.
+
+| Name | Description | Glyph |
+|-|-|-|
+| None | No glyph is displayed. The badge will appear blank or not be shown at all. | (No Badge Shown) |
+| Activity | A glyph indicating some form of activity is taking place within the app. | ![Screenshot](badge-activity.png) |
+| Alert | A glyph that suggests an alert or an important notification that may require immediate attention. | ![Screenshot](badge-alert.png) |
+| Alarm | A glyph that represents an alarm, possibly indicating a set reminder or a timed event. | ![Screenshot](badge-alarm.png) |
+| Attention | A glyph that signifies the need for attention, often used for notifications or new information. | ![Screenshot](badge-attention.png) |
+| Available | A glyph that indicates the user or a service is available, often used in communication apps. | ![Screenshot](badge-available.png) |
+| Away | A glyph that shows the user is away or inactive, commonly used in status indicators for communication apps. | ![Screenshot](badge-away.png) |
+| Busy | A glyph that represents the user being busy or engaged in an activity, preventing interruptions. | ![Screenshot](badge-busy.png) |
+| NewMessage | A glyph that indicates the arrival of a new message, often used in messaging and email applications. | ![Screenshot](badge-newmessage.png) |
+| Paused | A glyph that signifies a pause in activity or content, such as media playback being paused. | ![Screenshot](badge-paused.png) |
+| Playing | A glyph that indicates media or content is currently playing. | ![Screenshot](badge-playing.png) |
+| Unavailable | A glyph that shows the user or a service is not currently available or offline. | ![Screenshot](badge-unavailable.png) |
+| Error | A glyph that denotes an error has occurred, which may require user action to resolve. | ![Screenshot](badge-error.png) |
+
+These glyphs are system-provided and standardized, ensuring a consistent look and feel across different applications that use them. By using these glyphs, developers can convey specific states or notifications to users in a visually intuitive manner.
+
```

Comment:
Can add "You cannot provide your own badge image; only system-provided badge images can be used."

Reply:
Done